network-api/app/networkapi/people/apps.py

```diff
@@ -6,5 +6,4 @@ class PeopleConfig(AppConfig):
     verbose_name = 'people'
 
     def ready(self):
-        from networkapi.people.signals import setup_signals
-        setup_signals()
+        pass
```

Comment:
we should be able to remove this file entirely, if we remove the contents (but not the file) of network-api/app/networkapi/people/__init__.py